src/Dotenv.php

```diff
@@ -40,22 +40,41 @@ public function __construct($path, $file = '.env')
 
     /**
      * Load environment file in given directory.
+     * If .env file is missing, skip loading and return an empty array.
      *
      * @return array
      */
     public function load()
     {
+        if (file_exists($this->filePath)) {
```

Comment:
There is still a race condition here. The file could be deleted, or be inaccessible between this test, and actually getting the file. Please leave the existing behaviour of load as is, and have `safeLoad` be the method that suppressed exceptions.